common/metrics/defs.go

```diff
@@ -77,6 +77,10 @@ func NewBytesHistogramDef(name string, opts ...Option) histogramDefinition {
 	return histogramDefinition{globalRegistry.register(name, append(opts, WithUnit(Bytes))...)}
 }
 
+func NewMillisecondsHistogramDef(name string, opts ...Option) histogramDefinition {
```

Comment:
what's the difference between this and NewTimerDef?

Reply:
I didn't see NewTimerDef and don't actually know how this would be different; I'd wager our timers _are_ histograms. I'll change this to be a timer

Reply:
Changed